crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -162,11 +152,12 @@ impl KakarotSequencer {
     }
 
     pub fn compute_starknet_address(&self, evm_address: &Address) -> StateResult<ContractAddress> {
-        let base_class_hash = self.environment.base_account_class_hash.0;
-
-        let constructor_args = { vec![Felt::ONE, Felt::from_bytes_be_slice(&evm_address.0[..])] };
-
-        Ok(compute_starknet_address(evm_address, base_class_hash, &constructor_args).try_into()?)
+        Ok(compute_starknet_address(
```

Comment:
I think I prefer the version where we explicitly name the intermediate variables as it translates more what we are doing

Reply:
Sure just reverted but I also removed the useless usage of vec here